.github/workflows/publish.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow handles releasing all of the packages in samples.
+
+name: publish
+
+on: workflow_dispatch  # Manually trigger.
+
+permissions:
+  contents: write  # For checkout and tag.
+  packages: write  # For publish.
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    # Environment specific to releasing so we can isolate the npm token.
+    environment: release  
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Setup Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16
+
+    - name: NPM install
+      run: npm install
```

Comment:
i think this should probably be `npm ci` instead so it doesn't update dependencies during this step.

Reply:
Makes sense :D